### ISSUE TITLE
Add PrivateImageField

### ISSUE THREAD

**Issue by AndyLPK247** [edoburu/django-private-storage#4]
This is a request-for-enhancement: It would be really nice to have a PrivateImageField that borrows functionality from the standard ImageField class.

**Comment by vdboor** [comment]
Agreed, that would be a nice addition! A patch for this feature is more then welcome!

**Comment by kamranhossain** [comment]
This is really amazing package for me. Please let me know if PrivateImageField will add? Thanks @vdboor 
and @AndyLPK247 for your effort  :+1:

**Comment by asfaltboy** [comment]
btw, I defined this locally, is this what we're looking for?

```python

class PrivateImageField(PrivateFileField, models.ImageField):
    def __init__(self, *args, **kwargs):
        kwargs['max_file_size'] = kwargs.get(
            'max_file_size', settings.DEFAULT_MAX_FILE_SIZE
        )
        super().__init__(*args, **kwargs)
```

It works pretty well, but there are edge cases with regards to image transfer being required to get/set width/height attributes.
